### PR TITLE
6061 migrate org users list step 4

### PR DESCRIPTION
**Pull request by ThibaudAV**
**Issue**
gravitee-io/issues#6061

**Description**
- Add new component to create user avatar like IdentityPictureController
- Use this component in users list
- Set page layout

**Screenshots**

image
**Additional context**

### CODE REVIEW

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

Congrats, all the CI steps were 🟢 . The Bot was able to deploy the build of this PR, so you can now try it directly [here](https://apimnightlywebui24386.blob.core.windows.net/6061-migrate-org-users-list-step-4/index.html)! 🚀

_Notes_: The deployed app is linked to the management API of the Element Zero team's environment.

File: src/shared/components/gio-avatar/__snapshots__/gio-avatar.component.spec.ts.snap [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+`;
+
+exports[`GioAvatarComponent should display avatar image 1`] = `
+GioAvatarComponent {
+  "avatarContainerEleRef": ElementRef {
+    "nativeElement": <div
+      _ngcontent-a-c1=""
+      class="gio-avatar"
+    >
+      <svg
```
Comment (ThibaudAV) [inline src/shared/components/gio-avatar/__snapshots__/gio-avatar.component.spec.ts.snap:81]: :thi

File: src/shared/components/gio-avatar/gio-avatar.component.ts [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  @Input()
+  public id: string;
+  @Input()
+  public src: string;
+  @Input()
+  public name: string;
+  @Input()
+  public width: number;
+  @Input()
+  public borderRadius: boolean;
```
Comment (gaetanmaisse) [inline src/shared/components/gio-avatar/gio-avatar.component.ts:34]: To rename to `roundedBorder`?

**Review by gaetanmaisse — APPROVED** [review]

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

Congrats, all the CI steps were 🟢 . The Bot was able to deploy the build of this PR, so you can now try it directly [here](https://apimnightlywebui24386.blob.core.windows.net/6061-migrate-org-users-list-step-4/index.html)! 🚀

_Notes_: The deployed app is linked to the management API of the Element Zero team's environment.

**Pull request merged**